Menu/GameShell/Music Player/list_item.py

```diff
@@ -127,7 +128,7 @@ def Draw(self):
             
             if self._PlayingProcess > 0:
                 seek_posx = int(self._Width * self._PlayingProcess/100.0)
-                pygame.draw.line(self._Parent._CanvasHWND,(255,169,169),(self._PosX,self._PosY+self._Height-2),(self._PosX+seek_posx,self._PosY+self._Height-2),2)   
+                pygame.draw.line(self._Parent._CanvasHWND,SkinManager().GiveColor('Active'),(self._PosX,self._PosY+self._Height-2),(self._PosX+seek_posx,self._PosY+self._Height-2),2)   
```

Comment:
With any of these changed colors, I can add them back in. I just need to know what to name them :-)

It was 255,169,169
I used 175,90,0